pages/common/mutool.md

```diff
@@ -0,0 +1,23 @@
+# mutool
+
+> Convert PDF files, query information and extract data.
+
+- Convert pages 1-10 into 10 PNG images:
+
+`mutool convert -o {{image%d.png}} {{file.pdf}} {{1-10}}`
+
+- Convert pages 2, 3 and 5 of a PDF into text in the standard output:
+
+`mutool draw -F {{txt}} {{file.pdf}} {{2,3,5}}`
+
+- Concatenate two PDFs, skipping the first page of each:
```

Comment:
Can we change this example to just concatenate two pdfs ? I think that is the base version that we should mention. Rather than a complicated one.

Reply:
Sure! There's two options however:

* Use `1-N` syntax.
* Use no syntax at all, after all, it's optional.

Reply:
If it makes things simpler to explain, let us not use any syntax.